fragment-metadata.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Gallery of Glosses</title>
+    <link rel="stylesheet" href="css/gloss.css">
+    <link href="favicon.ico" rel="icon" type="image/x-icon" />
+    <script src="js/auth.js" type="module"></script>
+    <script src="js/deer.js" type="module"></script>
+    <script src="js/layout.js" type="module"></script>
+    <script src="js/text-witness-line-selector.js" type="module"></script>
+    <script src="js/bibliographicCitationView.js" type="module"></script>
+    <script src="js/glossModal.js" type="module"></script>
+    <script src="js/shared.js"></script>
+    <style>
+        input.required{
+            border: 1px solid black;
+        }
+        .pageContainer.is-toggled{
+            visibility: hidden;
+            height: 0px;
+        }
+        .loadingMsg{
+            color: white;
+            padding-top: 4em;
+        }
+        line:nth-child(2n) {
+            background-color: unset;
+        }
+        nav[user-input]{
+            margin-bottom: 1em;
+        }
+        nav[user-input] a{
+            cursor: pointer;
+        }
+        #reset{
+            cursor: pointer;
+        }
+        #needs{
+            margin-bottom: 3em;
+        }
+    </style>
+</head>
+
+<body class="container">
+    <gog-header class="container"></gog-header>
+    <h2 class="bg-basic"> Witness Fragment Metadata </h2>
+    <!--     
+    <p class="bg-light"> 
+        Connect Virtual Glosses With Real World Text 
+    </p> 
+-->
+    <p id="reset" class="bg-basic is-hidden">
+        This page automatically applied a text resource. If this isn't the text you want to use then <a onclick="startOver()">click here to start over</a>.
+    </p>
+    <p id="look" class="bg-light">
+        Witness Fragment Metadata
+    </p>
+    <div id="needs" class="row bg-basic">
+        <div class="col">
+            <p class="howTo"> 
+                It looks like you haven't provided the URI of the web resource with the text.  
+                If you are not using the <code>?witness-uri=</code> URL parameter then please choose a way to provide the text content. 
```

Comment:
I would be willing to (esp for MVP) say that fragment-metadata only applies to defined fragments and we don't need to create them within this space.

Reply:
That's acceptable.  The base functionality of generating a Witness assumes you are providing the source for that Witness.  A data structure change at that level would have to be a focus of its own; this closely resembles gloss-witness.html.  It's not clear to me where we are capture the text content that is the Witness anymore -- just on gloss-transcription.html and gloss-witness.html?

Reply:
Witness = Manuscript, so the text can be added on a manuscript-metadata.html page we'll invent. If you are on the other workflow, there is an imported block of text to start, so we can add it when we make the first instance of the Manuscript Witness